pacta/cj_build_tag.sh

```diff
@@ -0,0 +1,45 @@
+clones="PACTA_analysis create_interactive_report StressTestingModelDev pacta-data"
+url="git@github.com:2DegreesInvesting/"
+
+for repo in ${clones}
+do
+    remote="${url}${repo}.git"
+    git clone -b master ${remote} --depth 1
+    echo "--"
+done
```

Comment:
@cjyetman, to more clearly see what I changed see 
https://github.com/2DegreesInvesting/2diidockerrunner/pull/13/commits/3a8e5d85762f0e3aaf70eae2e8014efc487c2012

Reply:
awesome! thanks!